src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -125,7 +126,12 @@ const CreateConceptForm = props => (
         <div className="row col-12 custom-concept-list">
           <h6 className="text-left section-header">Names</h6>
           <CreateConceptTable {...props} />
-          <a href="#!" className="text-left add-more-names" id="add-more-name" onClick={props.handleNewName}>
+          <a
```

Comment:
This looks like a formatting change. Can this be done in a different PR?

Reply:
This is just fixing elslint warning.

Reply:
I reverted all the changes that are not part of this ticket.